managed-symbols-available/test.json

```diff
@@ -8,8 +8,7 @@
   "cleanup": true,
   "skipWhen": [
     "vmr-ci,version=6", // unpacking symbols is a hassle
-    "vmr-ci,version=7",
-    "non-packaged",
+    "non-packaged"
```

Comment:
FYI: we often leave the trailing comma - even though that is not allowed (along with the `//` comments) by strict JSON implementations - to make it easier to add items to the list without more diff noise.

Reply:
Noted, thanks!